### ISSUE TITLE
Eeducation additional fields

### ISSUE THREAD

**Issue by noraj** [saadq/resumake.io#159]
It feels like the education section is lacking of some fields:

- A free text-area field or an unordered list similar to the _Job Responsibilities_ in the Work section
- (Optional) Domain: domain of the degree (Examples of values: Information Security, Network & Telecommunication, Mathematics, Physics, Computer Science), but this could be redundant with the title in some case, so it's not the most important enhancement

Here is an example of Education section using both ideas:

![image](https://user-images.githubusercontent.com/16578570/109862777-220ecf80-7c61-11eb-8635-02cce278e673.png)

**Comment by saadq** [comment]
Yes, ultimately Education will be a lot like the Experiences section in the next version of Resumake which is under development. Custom sections are also supported so you can just make your own if you want to do your own customization for it as well

**Comment by noraj** [comment]
> Custom sections are also supported so you can just make your own if you want to do your own customization for it as well

In v3 right?

**Comment by saadq** [comment]
Yep
